.github/workflows/test.yml

```diff
@@ -0,0 +1,68 @@
+name: Test and upload Python package
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+    tags:
+      - 'v*'
+      - '!v*-dev'
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        # Test Zeek LTS, latest, and nightly, and
+        # don't fail for the nightly one.
+        include:
+          - repo: zeek
+            version: latest
+            experimental: false
+          - repo: zeek
+            version: lts
+            experimental: false
+          - repo: zeek-dev
+            version: latest
+            experimental: true
```

Comment:
Slight preference to using `continue_on_error` here so it's immediately clear what that bool stands for.